client/app/telecom/pack/xdsl/access/pack-xdsl-access.html

```diff
@@ -288,8 +290,30 @@ <h5 data-translate="xdsl_details_NRA_title"></h5>
                                     </dl>
                                 </div>
                             </li>
-
-
+                            <!--NRO-->
+                            <li class="oui-tile__item" data-ng-if="access.xdsl.isFiber">
+                                <div class="oui-tile__definition" data-ng-if="access.xdsl.isFiber">
+                                    <h5 data-translate="xdsl_details_NRO_title"></h5>
+                                    <dl>
+                                        <dt class="oui-tile__term"
+                                            data-translate="xdsl_details_NRO_name"></dt>
+                                        <dd class="oui-tile__description"
+                                            data-ng-bind="access.xdsl.nra"></dd>
+                                    </dl>
+                                </div>
+                            </li>
+                            <!--PTO (only for fiber)-->
+                            <li class="oui-tile__item" data-ng-if="access.xdsl.isFiber">
```

Comment:
This will already display only if `access.xdsl.isFiber`

Reply:
? No it won't